wiki/Ranking_Criteria/en.md

```diff
@@ -199,8 +199,8 @@ This category contains explicit allowance statements of concepts and rules that
   - **...have an average bit rate no lower than 128 kbps**, if such a source exists. Otherwise, use the highest quality available.
   - **...not be encoded upwards from a lower bitrate.**
 - **A beatmap may only contain one song file used by all difficulties.** Multiple song files within a single beatmap are unsupported and result in unexpected behaviour with preview times, metadata, etc.
-- **Beatmaps must be [hitsounded](/wiki/Beatmapping/Hitsound).** Hitnormals give feedback to the player, and additions (whistles, claps, and finishes) accent the most important parts of the music.
-  - **osu!mania beatmaps do not require hitsound additions.** This is to allow for easier approachability to osu!mania mappers of different upbringings. It is still highly recommended to add hitsounds to improve the feel of your beatmaps.
+- **Beatmaps must be [hitsounded](/wiki/Beatmapping/Hitsound).** Hitnormals give feedback to the player, and additions (whistles, claps, and finishes) accent the most important parts of the music.[^normal-vs-addition]
+  - **osu!mania beatmaps do not require hitsound additions[^normal-vs-addition].** This is to allow for easier approachability to osu!mania mappers of different upbringings. It is still highly recommended to use hitsound additions to improve the feel of your beatmaps.
```

Comment:
these come after punctuation

```suggestion
  - **osu!mania beatmaps do not require hitsound additions.**[^normal-vs-addition] This is to allow for easier approachability to osu!mania mappers of different upbringings. It is still highly recommended to use hitsound additions to improve the feel of your beatmaps.
```

Reply:
i put this one inside cuz its just for "hitsound additions", not whole sentence

Reply:
they come after punctuation anyway, with very few exceptions. we're following [wikipedia](https://en.m.wikipedia.org/wiki/Wikipedia:Manual_of_Style#Punctuation_and_footnotes) on this. see https://github.com/ppy/osu-wiki/pull/7391